app/error_messages.py

```diff
@@ -37,3 +37,5 @@
 VARIANTS_HAVE_MULTIPLE_QUESTION_TYPES = (
     "Variants have more than one question type for block."
 )
+NO_INTRODUCTION_BLOCK = "No introduction block for preview questions in schema"
```

Comment:
This should be more meaningful, maybe something like `No introduction block found. Introduction block is mandatory when using the preview questions feature.`

Reply:
Added.